examples/experimental/dagster-airlift/dagster_airlift/k8s/asset.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Mapping, Optional, Sequence
+
+from dagster import AssetExecutionContext, AssetSpec, Definitions, multi_asset
+from dagster_k8s import PipesK8sClient
+
+
+def k8s_pod_defs(
+    specs: Sequence[AssetSpec],
+    pod_spec_config: Mapping[str, Any],
+    pod_metadata: Mapping[str, Any],
+    in_cluster: bool = False,
+    cluster_context: Optional[str] = None,
+    config_file: Optional[str] = None,
+) -> Definitions:
```

Comment:
I think a key question here is:

should it be `k8s_op_operator_defs` or `k8s_pod_defs`.

`k8s_op_operator_defs`  clearly frames it as an _adapter_ layer. I think that is actually appropriate and it can be the place where people put all the airflow-specific weird mapping logic to making moving as easy as possible.

Reply:
With the current naming schema, I was taking my queue from the existing dbt_defs, which I guess is an imperfect analogy given that the corresponding airflow operator isn't a 1:1 mapping.

While I like the idea of making things as simple as possible for the user, I wonder if the reality will end up being more complex due to needing to support many versions of airflow, where argument types change over time. Doing things in this way kinda incurs that complexity up front once, as opposed to potentially hiding footguns in parameter mapping